### DISCUSSION TITLE
Technical co-founders wanted for a no-code AI agent builder business

### DISCUSSION THREAD

**Discussion by smy0797** [infiniflow/ragflow#2179] [General]
Hi All,

I am an entrepreneur based in London, UK; I am looking for two technical co-founders (described below) to build a no-code AI agent builder based on the RAGflow project, like Bubble.io for AI chatbots and Agents.

One to modify and enhance RAGFlow existing functions
The other one to work on external integrations e.g. publishing agents on third-party webs, WhatsApp
Please DM me if anyone is interested. My email: [mysong0797@gmail.com](mailto:mysong0797@gmail.com)

Hi admins, if this is the wrong place to ask such a question, please feel free to move my post.

Thanks
M.

**Comment by yingfeng** [comment]
Hi, just here is OK. You could also post to discord. RAGFlow is committed to building an open ecosystem that makes enterprise-grade LLM available to as many businesses as possible.
